bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_Controller.cpp

```diff
@@ -64,6 +64,13 @@ void Controller_Trampoline::reinit()
 void Controller_Trampoline::callScriptMethod(
         const py::object& self, Event* event, const std::string & methodName)
 {
+    if(f_printLog.getValue())
+    {
+        std::string name = std::string("on")+event->getClassName();
+        std::string eventStr = py::str(PythonFactory::toPython(event));
+        msg_info() << name << " " << eventStr;
```

Comment:
```suggestion
        dmsg_info() << name << " " << eventStr;
```
because the message looks like targeted for developers

Reply:
Thanks for the suggestion but no. The aims of that is to allow people that are making a scene (not c++ sofa developper).

Reply:
To print something in python you need to override a method with this name convention onXXXXXX where XXXXXX is the name of the event. 

You don't necesserely know which event's names is emitted (because this depend on which part component loops are in the graph). 

Visualising the raw events is actually really useful and I often override the  "onEvent" method just for printing the event's name, before actually writing onXXXXX (and then remove the onEvent). This workflow is in-efficient and checking printLog box is much easier. Despite printLog's purpose is actually to activate the printing of extra-info it may be debatable to use a second data field like "showRawEvents" instead.